ios/FluentUI.Demo/FluentUI.Demo.xcodeproj/project.pbxproj

```diff
@@ -14,6 +14,7 @@
 		7D23482A24D89C1C00FBE057 /* AvatarGroupViewDemoController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 7D23482924D89C1C00FBE057 /* AvatarGroupViewDemoController.swift */; };
 		7DC2FB2B24C0F4FD00367A55 /* TableViewCellFileAccessoryViewDemoController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 7DC2FB2A24C0F4FD00367A55 /* TableViewCellFileAccessoryViewDemoController.swift */; };
 		8AD4FF842493F2500085C076 /* ContactViewDemoController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 8AD4FF832493F2500085C076 /* ContactViewDemoController.swift */; };
```

Comment:
contactviewdemocontroller.swift itself is not interesting. the toolkit component is meant to be horizontal scrollable. if you are creating contactcollectionviewdemocontroller separately, I would delete the other one.

Reply:
I'm a bit unclear on what you mean. Are you saying we should remove the `ContactViewDemoController.swift` and any references to it since the "full control" should really be scrollable? (Therefore we only have `ContactCollectionViewDemoController.swift`?)

Reply:
Resolved via messages. Will be removing `ContactViewDemoController.swift` and any references to it.